### ISSUE TITLE
Achievements

### ISSUE THREAD

**Issue by nbw** [battlesnakeio/roadmap#133]
We could add tracking to the game and give teams some feedback on their achievements, but also something for the presenters to use on the day of like:

* who's killed the most snakes?
* who's snake at the most food?
* survived the longest (not necessarily at the most)
* etc.

We'd just need to record a few key points, maybe send a sum at the end of the game like total kills, snake length, turn number, etc.

**Comment by codeallthethingz** [comment]
I agree with the tracking and the interesting things to talk about for the commentators. 

I think calling them achievements implies that the thing they're doing is the correct thing, when in fact we don't know whether long snakes or short snakes do bette.  Giving an achievement for long snakes might be a nudge in the wrong direction. 

@dlsteuer In the engine I think we should track these and then return them with the last frame?  or have a stats endpoint for a game?

- each snake length
- number of turns each snake made
- amount of food eaten for each snake
- how the each of the snakes died

**Comment by dlsteuer** [comment]
I think this needs to live in play, and is part of a larger stats project, that feeds into achievements.
